blazor/diagram/accessibility.md

```diff
@@ -0,0 +1,86 @@
+---
+layout: post
+title: Accessibility in Blazor Diagram Component | Syncfusion
+description: Checkout and learn here all about accessibility in Syncfusion Blazor Diagram component and much more details.
+platform: Blazor
+control: Diagram 
+documentation: ug
+---
+
+# Accessibility in Blazor Diagram Component
+
+Accessibility is achieved in the diagram component through WAI-ARIA standard and keyboard navigation support. The diagram features can be effectively accessed through assistive technologies such as screen readers.
+
+The accessibility compliance for the diagram component is outlined below.
+
+* [Color Contrast](../common/accessibility#color-contrast)
+* [Mobile Device Support](../common/accessibility#mobile-device-support)
+* [Keyboard Navigation Support](../common/accessibility#keyboard-navigation-support)
+* [Axe-core Accessibility Validation](../common/accessibility#ensuring-accessibility)
+
+## WAI-ARIA
+
+WAI-ARIA (Accessibility Initiative – Accessible Rich Internet Applications) defines a way to increase the accessibility of web pages, dynamic content, and user interface components developed with HTML, JavaScript,and related technologies. ARIA provides additional semantics to describe the role, state, and functionality of web components.
+
+The following ARIA attributes are used in the diagram component:
+
+* aria-label (attribute)
+
```

Comment:
WAI-ARIA (Web Accessibility Initiative – Accessible Rich Internet Applications) defines standards to enhance the accessibility of web pages, dynamic content, and user interface components developed with HTML, JavaScript, and related technologies. ARIA supplements the semantics of web components by providing additional information about their role, state, and functionality.

The diagram component utilizes the following ARIA attribute:

aria-label (attribute): Provides a label or description for an element to aid assistive technologies in conveying its purpose or meaning to users with disabilities.

Reply:
improve documentation have updated in the below link

https://blazor.syncfusion.com/documentation/datagrid/accessibility